api/v1alpha1/selfnoderemediationconfig_webhook_test.go

```diff
@@ -119,6 +119,34 @@ var _ = Describe("SelfNodeRemediationConfig Validation", func() {
 
 	})
 
+	Describe("deleting SelfNodeRemediationConfig CR", func() {
+		var conf *SelfNodeRemediationConfig
+		BeforeEach(func() {
+			conf = createDefaultSelfNodeRemediationConfigCR()
```

Comment:
It seems a bit misleading. The function is called `createDefaultSelfNodeRemediationConfigCR` but it doesn't create the "default" SNRC (i.e. the name is "test").

It might be better to use the default name in the function, and then change it to something else only when necessary (e.g. in "SelfNodeRemediationConfig CR is non default").

Reply:
> but it doesn't create the "default" SNRC

Good point !
I've removed the "default" from the method name, I think I prefer this over the alternative of changing the name.
Let me know if that's ok.

Reply:
I think I'd prefer something like `CreateTestSelfNodeRemediationConfigCR` then, or making the name a input, for clarity